pq-crypto/kyber_90s_r2/aes_c.c

```diff
@@ -530,10 +530,14 @@ static inline uint32_t UINT32_TO_BE(const uint32_t x) {
 		uint32_t val;
 		uint8_t bytes[4];
 	} y;
+
+	/* As part of the union, these bytes get read when y.val is read */
 	y.bytes[0] = (x >> 24) & 0xFF;
 	y.bytes[1] = (x >> 16) & 0xFF;
 	y.bytes[2] = (x >> 8) & 0xFF;
+    /* cppcheck-suppress unreadVariable */
```

Comment:
indent ;)

Reply:
Indents actually fixed for real this time *fingers crossed*